### PR TITLE
Bug 1377807 - Move memcached options from data/params to localconfig

### PR DESCRIPTION
**Pull request by umohm1**

### CODE REVIEW

File: template/en/default/setup/strings.txt.pl [hunk 1]
```diff
@@ -387,6 +387,15 @@ END
 WARNING: Dependency graph images are not accessible.
 Delete ##dir##/.htaccess and re-run checksetup.pl.
 END
+    memcached_servers => <<'END',
+If this option is set, Bugzilla will integrate with `Memcached
+<http://www.memcached.org/>`_. Specify one or more servers, separated by
```
Comment (floatingatoll) [inline template/en/default/setup/strings.txt.pl]: Please remove the `_` character here.

File: template/en/default/setup/strings.txt.pl [hunk 2]
```diff
@@ -54,7 +54,7 @@ END
 to continue.
 END
     cpanfile_created => "##file## created",
-    cpan_bugzilla_home =>
```
Comment (floatingatoll) [inline template/en/default/setup/strings.txt.pl]: I had a hard time finding the actual code changes due to the sheer volume of unrelated whitespace fixes in this pull request. Please consider separating them into their own whitespace-only commit and/or pull request in the future.

File: template/en/default/setup/strings.txt.pl (hunk 1 above)
Comment (dylanwh) [inline template/en/default/setup/strings.txt.pl]: yeah, the _ is string concatenation in the templates, but it isn't needed here.

File: template/en/default/setup/strings.txt.pl [hunk 3]
```diff
@@ -387,6 +387,15 @@ END
 WARNING: Dependency graph images are not accessible.
 Delete ##dir##/.htaccess and re-run checksetup.pl.
 END
+    memcached_servers => <<'END',
```
Comment (dylanwh) [inline template/en/default/setup/strings.txt.pl]: these should be named localconfig_memcached_servers and localconfig_memcached_namespace and be moved up near the other options -- possibly in alphabetical order

**Pull request merged**